### PR TITLE
development

### PR DESCRIPTION
**Pull request by Andrew2831**
Added all configurations, created dockerfile, configured linting and testing with ci

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# MTRPZ LAB 4
```
Comment (stbestichhh) [inline README.md:1]: where is content?

File: src/api/dto/updateRecepie.dto.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import { CreateRecepieDto } from "./createRecepie.dto";
+
+export interface UpdateRecepieDto extends Partial<CreateRecepieDto> {}
```
Comment (stbestichhh) [inline src/api/dto/updateRecepie.dto.ts:3]: good

File: src/api/recepies.service.ts [hunk 3]
```diff
@@ -0,0 +1,120 @@
+import { handleErrorSync } from '@stlib/utils';
+import { RecepieEntity, RecepieType } from '../db';
+import { CreateRecepieDto, UpdateRecepieDto } from './dto';
+import { Response, Request } from 'express';
+import { v4 as uuidv4 } from 'uuid';
+
+export class RecepiesService {
+  private recepieRespository: typeof RecepieEntity = RecepieEntity;
```
Comment (stbestichhh) [inline src/api/recepies.service.ts]: what for is that?????, just use this entity, not need to create new variable

File: src/api/recepies.service.ts [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+        name: dto.name,
+        ingridients: dto.ingridients,
+        steps: dto.steps,
+        notes: dto.notes
+      });
+
+      return res.status(201).json({ recepie });
+    } catch (e) {
+      handleErrorSync(e);
+      return res.json('Error occured, see console logs.')
```
Comment (stbestichhh) [inline src/api/recepies.service.ts]: where is status?

File: src/db/db.ts [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import { handleErrorSync, options } from "@stlib/utils";
+import path from "path";
+import { Sequelize } from "sequelize";
+
+export const sequelize = new Sequelize({
+  dialect: 'sqlite',
+  storage: path.join(__dirname, 'recepies.db'),
+  logging: Boolean(options?.log).valueOf(),
```
Comment (stbestichhh) [inline src/db/db.ts:8]: thx for using my library)

File: src/db/recepie.entity.ts [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+      allowNull: false,
+    },
+    notes: {
+      type: DataTypes.STRING,
+      allowNull: true,
+    },
+  },
+  {
+    tableName: 'recepies',
+    timestamps: false,
```
Comment (stbestichhh) [inline src/db/recepie.entity.ts]: why false?

File: test/index.spec.ts [hunk 7]
```diff
@@ -0,0 +1,72 @@
+import supertest from 'supertest';
+import { sequelize } from '../src/db';
+import { app } from '../src/server';
+import { CreateRecepieDto, UpdateRecepieDto } from '../src/api/dto';
+
+describe('Recepies', () => {
+  let id: string = '';
+  let name: string = '';
```
Comment (stbestichhh) [inline test/index.spec.ts]: just save a recepie as object, why so many vars?)

**Review by stbestichhh — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Andrew2831) [reply README.md:1]: will do it later

File: src/api/recepies.service.ts (hunk 3 above)
Comment (Andrew2831) [reply src/api/recepies.service.ts]: fixed

File: src/api/recepies.service.ts (hunk 4 above)
Comment (Andrew2831) [reply src/api/recepies.service.ts]: fixed

File: src/db/recepie.entity.ts (hunk 6 above)
Comment (Andrew2831) [reply src/db/recepie.entity.ts]: fixed

File: test/index.spec.ts (hunk 7 above)
Comment (Andrew2831) [reply test/index.spec.ts]: fixed

**Review by stbestichhh — APPROVED** [review]

**Pull request merged**